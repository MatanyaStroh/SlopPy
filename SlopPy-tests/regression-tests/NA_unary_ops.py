# do a bunch of unary operations on an NA object

x = 1 / 0
assert type(x) is NA

assert type(-x) is NA
assert type(+x) is NA
assert type(abs(x)) is NA
assert type(~x) is NA

