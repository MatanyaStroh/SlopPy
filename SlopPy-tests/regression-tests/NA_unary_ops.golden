{NA <type 'exceptions.ZeroDivisionError'> 'integer division or modulo by zero'
 Traceback (most recent call last):
  File "regression-tests/NA_unary_ops.py", line 3, in <module>
    x = 1 / 0

Locals: {'__builtins__': <module '__builtin__' (built-in)>, '__name__': '__main__', '__file__': 'regression-tests/NA_unary_ops.py', '__doc__': None, '__package__': None}
}
{NA_unary | regression-tests/NA_unary_ops.py <module>:6}
{NA_unary | regression-tests/NA_unary_ops.py <module>:7}
{NA_unary | regression-tests/NA_unary_ops.py <module>:8}
{NA_unary | regression-tests/NA_unary_ops.py <module>:9}
